src/DecayTree.cxx

```diff
@@ -205,4 +205,14 @@ const DecayTreeVector select_changed(const DecayTreeVector& dtv)
     return C;
 }
 
+//-------------------------
+const DecayTreeVector daughter_decay_trees(const DecayTree& dt)
```

Comment:
Superfluous, since we have `DecayTree::daughterDecayTrees()` --- plus the order and content are not guaranteed. Should be removed.